deploy/gppkg/madlib.spec.in

```diff
@@ -7,7 +7,7 @@
 BuildRoot:      @MADLIB_GPPKG_RPM_SOURCE_DIR@
 Summary:        Apache MADlib for @GPDB_VARIANT@ Database
 License:        @CPACK_RPM_PACKAGE_LICENSE@
-Name:           Apache_MADlib
+Name:           madlib
```

Comment:
This change in name created some problems when trying to upgrade from 1.11 to 1.12. It was changed with the 1.11 release and it was not apparent until now. The rpm will be renamed to `apache-madlib-1.12-bin-Linux.rpm` as usual.